runtime/themes/jetbrains_dark.toml

```diff
@@ -0,0 +1,96 @@
+"type.builtin" = "red207"
+"type.enum" = "red199"
+"constant.builtin" = "red207"
+"constant.numeric" = "blue184"
+string = "green171"
+comment = "blue133"
+"comment.block.documentation" = "green130"
+"variable.builtin" = "red207"
+"variable.other.member" = "red199"
+label = "red199"
+keyword = "red207"
+function = "blue245"
+"function.macro" = "red179"
+tag = "red213"
+
+"markup.heading" = { fg = "red199", modifiers = ["italic"] }
+"markup.heading.marker" = "red207"
+"markup.list" = "red207"
+"markup.bold" = { modifiers = ["bold"] }
+"markup.italic" = { modifiers = ["italic"] }
+"markup.strikethrough" = { modifiers = ["crossed_out"] }
+"markup.link" = { fg = "blue247", underline = { style = "line" } }
+"markup.raw" = { bg = "blue64" }
+
+"diff.plus" = "green145"
+"diff.minus" = "blue145"
+"diff.delta" = "blue173"
+
+"ui.background" = { fg = "blue196", bg = "blue34" }
+"ui.gutter" = "blue56"
+"ui.gutter.selected" = { fg ="blue56", bg = "blue46" }
+"ui.linenr" = "blue89"
+"ui.linenr.selected" = "blue171"
+"ui.statusline" = { bg = "blue48" }
+"ui.statusline.normal" = { fg = "blue48", bg = "green156", modifiers = ["bold"] }
+"ui.statusline.insert" = { fg = "blue48", bg = "blue247", modifiers = ["bold"] }
+"ui.statusline.select" = { fg = "blue48", bg = "red194", modifiers = ["bold"] }
+"ui.popup" = { bg = "blue48" }
+"ui.text.focus" = "blue247"
+"ui.virtual.whitespace" = "blue122"
+"ui.virtual.indent-guide" = "blue56"
+"ui.virtual.inlay-hint" = { fg = "blue145", bg = "blue50" }
```

Comment:
You may want to set a `"ui.virtual" = { fg = "<something>" }` (some similar value to what you have here) to cover all the features you're not explicitly theming here.

Also you should add a `"ui.virtual.ruler" = { bg = "<something>" }` so that rulers show up. You can use `:set rulers [80]` to turn on rulers temporarily

Reply:
Both keys have now been added.